DOC: replace "-c intel" in doc files

bdmoore1 opened the pull request:
Change references to Anaconda to now point to the Intel repository.

### Description 
Anaconda license is no longer supported. The package has been moved to an Intel repository. Rodriguo Bernal Castro verified that 'intel' as a package should be changed to **https://software.repos.intel.com/python/conda/**

Fixes # - _issue number(s) if exists_

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
- [ ] The unit tests pass successfully.
- [ ] I have run it locally and tested the changes extensively.
- [ ] I have resolved any merge conflicts that might occur with the base branch.
- [ ] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
- [ ] I have added a respective label(s) to PR if I have a permission for that.

samir-nasibli commented:
> Change references to Anaconda to now point to the Intel repository.
> 
> ### Description
> Anaconda license is no longer supported. The package has been moved to an Intel repository. Rodriguo Bernal Castro verified that 'intel' as a package should be changed to **https://software.repos.intel.com/python/conda/**
> 
> Fixes # - _issue number(s) if exists_
> 
> * [x]  I have reviewed my changes thoroughly before submitting this pull request.
> * [ ]  I have commented my code, particularly in hard-to-understand areas.
> * [x]  I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
> * [ ]  The unit tests pass successfully.
> * [ ]  I have run it locally and tested the changes extensively.
> * [ ]  I have resolved any merge conflicts that might occur with the base branch.
> * [ ]  Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
> * [ ]  I have added a respective label(s) to PR if I have a permission for that.

Hi @bdmoore1 thank you for a such replace.
Following this I guess we have doc files where it should be updates as well.
https://github.com/intel/scikit-learn-intelex/blob/main/daal4py/INSTALL.md
https://github.com/intel/scikit-learn-intelex/blob/main/daal4py/README.md
https://github.com/intel/scikit-learn-intelex/blob/main/doc/sources/oneapi-gpu.rst
https://github.com/intel/scikit-learn-intelex/blob/main/doc/sources/quick-start.rst

bdmoore1 commented:
> > Change references to Anaconda to now point to the Intel repository.
> > ### Description
> > Anaconda license is no longer supported. The package has been moved to an Intel repository. Rodriguo Bernal Castro verified that 'intel' as a package should be changed to **https://software.repos.intel.com/python/conda/**
> > Fixes # - _issue number(s) if exists_
> > 
> > * [x]   I have reviewed my changes thoroughly before submitting this pull request.
> > * [ ]   I have commented my code, particularly in hard-to-understand areas.
> > * [x]   I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
> > * [ ]   The unit tests pass successfully.
> > * [ ]   I have run it locally and tested the changes extensively.
> > * [ ]   I have resolved any merge conflicts that might occur with the base branch.
> > * [ ]   Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
> > * [ ]   I have added a respective label(s) to PR if I have a permission for that.
> 
> Hi @bdmoore1 thank you for a such replace. Following this I guess we have doc files where it should be updates as well. https://github.com/intel/scikit-learn-intelex/blob/main/daal4py/INSTALL.md https://github.com/intel/scikit-learn-intelex/blob/main/daal4py/README.md https://github.com/intel/scikit-learn-intelex/blob/main/doc/sources/oneapi-gpu.rst https://github.com/intel/scikit-learn-intelex/blob/main/doc/sources/quick-start.rst

Yes, those should be updated as well. I will work on that next week when I return from vacation.
